core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/TenantConstants.java

```diff
@@ -61,6 +61,7 @@
     public static final String IS_TENANT_ACTIVE_SQL = "SELECT UM_ACTIVE FROM UM_TENANT WHERE UM_ID=?";
     public static final String DELETE_TENANT_SQL = "DELETE FROM UM_TENANT WHERE UM_ID=?";
     public static final String DELETE_TENANT_BY_UUID_SQL = "DELETE FROM UM_TENANT WHERE UM_TENANT_UUID=?";
+    public static final String DELETE_TENANT_DOMAIN_BY_TENANT_ID_SQL = "DELETE FROM UM_DOMAIN WHERE UM_TENANT_ID=?";
```

Comment:
Lets make this more meaningful since we are trying to clear domains

Reply:
fixed 775da9a1e009be12698bc9ac9c86a8491b975b16